code/rust-sokoban-c04-01/src/components.rs

```diff
@@ -0,0 +1,103 @@
+use specs::{Component, NullStorage, VecStorage, World, WorldExt};
```

Comment:
could you please rebase to make sure the diff with master is correct? I assume these changes are unrelated to the translation :)